### ISSUE TITLE
Warning vs. error when a certain feature is not supported

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-mod-braille#80]
For example when CSS is valid but not supported on the system. What is better in this case? Show a warning or abort the conversion with an error?

**Comment by josteinaj** [comment]
I'd say always display a warning, never fail or silently ignore.

**Comment by bertfrees** [comment]
Agree. Problem with warnings right now is that they don't actually end up in the user interface. I'm going to address that in phase 3.
